docs/guides/authjs-migration.mdx

```diff
@@ -137,13 +137,13 @@ With Clerk, you can control access to your application in a few different ways.
 
 #### Control access to your app with Clerk Middleware
 
-You will need to remove the Auth.js Middleware from your application, and replace it with the Clerk Middleware helper. Auth.js's middleware always rejects unauthorized requests. You may have additionally configured the Next.js middleware config to protect specific private routes. You will need to make note of this configuration so you can recreate it.
+You will need to remove the Auth.js Middleware from your application, and replace it with the Clerk's Middleware helper, `authMiddleware()`. 
 
-Remove the Auth.js Middleware and replace it with Clerk's `authMiddleware()`.
+Auth.js's middleware always rejects unauthorized requests. You may have additionally configured the Next.js middleware config to protect specific private routes. You will need to make note of this configuration so you can recreate it.
 
-Clerk's Middleware gives you fine-grained control over handling the authenticated state and will, by default, run for your entire application. 
+Clerk's Middleware gives you fine-grained control over handling the authenticated state and will, by default, run for your entire application.
 
-The example below is a basic configuration that protects all routes. You will need to edit this to allow specific routes to be public as needed. Read the [`authMiddleware()`](/docs/references/nextjs/auth-middleware) documentation to learn more about how you can configure your Middleware.
+The example below is a basic configuration that requires authentication for all routes. You will need to edit this to allow specific routes to be public, as needed. Read the [`authMiddleware()`](/docs/references/nextjs/auth-middleware) documentation to learn more about how you can configure your Middleware.
```

Comment:
"requires authentication for" isn't strictly true for Auth.js. It can work differently.

Reply:
but its for the authMiddleware example, not Auth.js

Reply:
My bad. Too much time staring at this guide. Change applied.